src/style.css

```diff
@@ -1,3 +1,114 @@
 body {
   margin: 0;
+  font-family: "Roboto", arial, sans-serif;
+}
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 0 40px;
+}
+
+.card__title {
+  margin: 0 16px 4px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector. (checklist - point 7)